src/components/PrimaryPanes/SourcesTree.js

```diff
@@ -301,7 +302,7 @@ class SourcesTree extends Component<Props, State> {
   }
 
   render() {
-    const { setExpandedState, expanded } = this.props;
+    const expanded = this.props.expanded;
```

Comment:
I don't get this change. You could have left it and made `onExpand/onCollapse` call it without `this.props`

Reply:
The change was necessary after the implicit import of `setExpandedState` could have also renamed it, I guess.